spelling_wordlist.txt

```diff
@@ -37,7 +38,9 @@ changelog
 checkmark
 codebase
 config
-conjuncted
+conjoined
+datastructure
```

Comment:
I would just write "data structure" instead of adding this to the dictionary

Reply:
good point; changing